containers/ReportPage.js

```diff
@@ -344,6 +343,22 @@ class ReportPage extends React.Component {
     return name;
   }
 
+  findOrgCodeFilter() {
+    const rolesToSearchIn = [ 'editStops', '' ];
+    let orgCodeFilter = null;
+    let userRoles = JSON.parse(this.props.code);
```

Comment:
`this.props.code` should be `this.props.orgCode`?
This code can be null. This will result in a nasty exception when trying to parse it.

Reply:
The code is passed to `findOrgCodeFilter` as a parameter, but by the looks of the signature, it doesn't take an argument

Reply:
` if (firstOrgFound)` will suffice, checking both null and `typeof === undefined`.

Reply:
Didn't get that last one (checking undefined)